tests/test_platform.rb

```diff
@@ -221,6 +221,10 @@ def test_fans
   end
 
   def test_virtual_services
+    # Only run this test if a virtual-service is installed
+    if @device.cmd('show virtual-service global')[/services installed : 0$/]
```

Comment:
1. This test should have a `validate_property_excluded?` check
2. nit: You can can use `config()` instead of `@device.cmd()` to do the same thing.